src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder res = new StringBuilder("");
```

Comment:
res is not very informative 
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Reply:
done